### ISSUE TITLE
What is this "Annotation SYNC" do?

### ISSUE THREAD

**Issue by jpdup** [MylioExternal/bugs#26]
I searched online, I clicked it on/off can't figure what the hell this critter does

![image](https://user-images.githubusercontent.com/2709213/48604291-3878ac00-e92e-11e8-8c20-2144b0884592.png)

**Comment by jpdup** [comment]
Is that the OLD "Show TPO",  if that's so I do not see the TPO button on the folder

**Comment by mylodeon** [comment]
Yes, it enables the old TTS. You also have to have "Selective Sync" enabled on a device for the TTS buttons to show up.

It's a bug (I just filed #37103) that we show that menu item in the first place if you don't have selective sync enabled.

**Comment by jpdup** [comment]
In that case why is it offering me to Annotate menu choice in the pull down menu,  if the device is not "setup" for TTS (Selective Sync)

**Comment by mylodeon** [comment]
Yes, that's the bug I mentioned above (37103).
